### PR TITLE
libkb: don't use `poll=10` when loading upaks for chat, etc

### PR DESCRIPTION
**Pull request by maxtaco**
I'm trying to reduce the DB load of that polling query, and it bubbles all the way back to the client. It would be nice to disable server polling in more places, but a lot of tests depend upon it. Sigh.

### CODE REVIEW

File: go/engine/merkle_audit.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ var (
 var MerkleAuditSettings = BackgroundTaskSettings{
 	Start:        5 * time.Minute,
 	StartStagger: 1 * time.Hour,
-	Interval:     6 * time.Hour,
+	Interval:     24 * time.Hour,
```
Comment (heronhaye) [inline go/engine/merkle_audit.go]: Just wondering, do you know how this works on mobile? Is it 24 hours real time or amount of time in foreground?

Comment (maxtaco) [reply go/engine/merkle_audit.go]: Good Q, I don't know... @pzduniak ?

Comment (maxtaco) [reply go/engine/merkle_audit.go]: Seems to only run on wifi

Comment (heronhaye) [reply go/engine/merkle_audit.go]: I think josh did that for the box auditor too a while back.

Comment (pzduniak) [reply go/engine/merkle_audit.go]: >do you know how this works on mobile?
there should be an early return if it detects its not running on desktop, unless someone changed it

Comment (heronhaye) [reply go/engine/merkle_audit.go]: ok I asked miles, he says timer is real time

**Pull request merged**